validator/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface ValidatorCtx {
+  // This key must be hot and used for on-demand signing
+  publicKeystore: string;
+  // Password for public keystore
+  publicKeystorePassword: string;
+  // This key can be kept cold and is used upon exit
+  withdrawalKeystore: string;
+  // Password for withdrawal keystore
+  withdrawalKeystorePassword: string;
+  // Connection string to RPC
+  rpcUrl: string;
+}
+export const ValidatorCtx = {
```

Comment:
We probably don't need this const. We only need const types for things that we're planning on ssz-ing.

Reply:
that makes a  lot of sense